hugo/layouts/sitemap.xml

```diff
@@ -0,0 +1,46 @@
+{{ printf "<?xml version=\"1.0\" encoding=\"utf-8\" standalone=\"yes\" ?>" | safeHTML }}
+<urlset xmlns="http://www.sitemaps.org/schemas/sitemap/0.9"
+  xmlns:xhtml="http://www.w3.org/1999/xhtml">
+  {{ range .Data.Pages }}
+  <url>
+    <loc>{{ .Permalink }}</loc>{{ if not .Lastmod.IsZero }}
+    <lastmod>{{ safeHTML ( .Lastmod.Format "2006-01-02T15:04:05-07:00" ) }}</lastmod>{{ end }}{{ with .Sitemap.ChangeFreq }}
+    <changefreq>{{ . }}</changefreq>{{ end }}{{ if ge .Sitemap.Priority 0.0 }}
+    <priority>{{ .Sitemap.Priority }}</priority>{{ end }}
+  </url>
+  {{ end }}
+  {{ range .Site.Data.volumes }}
+    {{ with .pdf }}
+      {{ if (hasPrefix . "https://www.aclweb.org/anthology") }}
+  <url><loc>{{ . }}</loc></url>
+      {{ end }}
+    {{ end }}
+  {{ end }}
+  {{ range .Site.Data.papers }}
+    {{ range . }}
+      {{ with .pdf }}
+        {{ if (hasPrefix . "https://www.aclweb.org/anthology") }}
+  <url><loc>{{ . }}</loc></url>
+        {{ end }}
+      {{ end }}
+      {{ with .attachment }}
+        {{ range . }}
+          {{ if (hasPrefix .url "https://www.aclweb.org/anthology") }}
+  <url><loc>{{ .url }}</loc></url>
+          {{ end }}
+        {{ end }}
+      {{ end }}
+      {{ with .revision }}
+        {{ range . }}
+  <url><loc>{{ .url }}</loc></url>
```

Comment:
Potential edge case: Can there ever be revisions for externally hosted papers?

Reply:
Maybe, but we wouldn't know about them. I don't think there are any now.